docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -698,6 +698,100 @@ In both cases, importing those modules will allow HTTP message bodies to be read
 and serialised to JSON, for <<resource-types,all the types not already registered with a more specific
 serialisation>>.
 
+==== Advanced Jackson-specific features
+
+When using the `quarkus-resteasy-reactive-jackson` extension there are some advanced features that RESTEasy Reactive supports.
+
+===== @JsonView support
+
+JAX-RS methods can be annotated with https://fasterxml.github.io/jackson-annotations/javadoc/2.10/com/fasterxml/jackson/annotation/JsonView.html[@JsonView]
+in order to customize the serialization of the returned POJO, on a per method-basis. This is best explained with an example.
+
+Let's assume we have the following POJO:
+
+[source,java]
+----
+public class User {
+
+    @JsonView(Views.Private.class)
+    public int id;
+
+    @JsonView(Views.Public.class)
+    public String name;
+}
+----
+
+A typical use of `@JsonView` is to hide certain fields on certain methods. In that vein, let's define two views:
+
+[source,java]
+----
+public class Views {
+
+    public static class Public {
+    }
+
+    public static class Private extends Public {
+    }
+}
+----
```

Comment:
Shouldn't you declare the views before using them?

Reply:
Good point, I'll update

Reply:
Fixed